bandwidth/tests/test_bxml.py

```diff
@@ -408,4 +408,27 @@ def test_generate_transfer_bxml_verb(self):
             'asdf', 'c-93d6f3c0-be584596-0b74-4fa2-8015-d8ede84bd1a4')
         assert actual == expected
     
+    def test_start_and_stop_stream_bxml_verbs(self):
+        start_stream_expected = '<?xml version="1.0" encoding="UTF-8"?><Response><StartStream destination="https://www/test/com/stream" name="test_stream" streamEventUrl="https://www.test.com/event" streamEventMethod="POST" username="username" password="password"/></Response>'
+        start_stream_response = Response()
+        start_stream = StartStream(
+            destination='https://www/test/com/stream',
+            name='test_stream',
+            streamEventUrl='https://www.test.com/event',
+            streamEventMethod='POST',
+            username='username',
+            password='password'
+        )
+        start_stream_response.add_verb(start_stream)
```

Comment:
+1 for adding Unit Test.

Couple of things:

- The single test can be separated into two  one for `start_stream` and other for `stop_stream` 
- The property values can be extracted to their own variables and then both can be referenced in `expected` and `actual` objects with a single source of truth